execution/engine/emr_engine.go

```diff
@@ -172,6 +172,17 @@ func (emr *EMRExecutionEngine) Execute(executable state.Executable, run state.Ru
 }
 
 func (emr *EMRExecutionEngine) generateApplicationConf(executable state.Executable, run state.Run, manager state.Manager) []*emrcontainers.Configuration {
+	// Define custom tags
+	customTags := fmt.Sprintf("[\"flotilla_run_id:%s\", \"team:%s\", \"kube_workflow:%s\", \"kube_task_name:%s\"]",
```

Comment:
Backticks can be used so that quotes don't need to be escaped
```suggestion
	customTags := fmt.Sprintf(`["flotilla_run_id:%s", "team:%s", "kube_workflow:%s", "kube_task_name:%s"]`,
```